docs/sql-ref-identifier.md

```diff
@@ -64,6 +64,7 @@ Note: If `spark.sql.ansi.enabled` is set to true, ANSI SQL reserved keywords can
 {% highlight sql %}
 -- This CREATE TABLE fails with ParseException because of the illegal identifier name a.b
 CREATE TABLE test (a.b int);
+-- output
```

Comment:
This comment looks fine, but I think we need the same comment in the other error output.

Reply:
Maybe have the same format as ansi compliance page?
<img width="805" alt="Screen Shot 2020-05-01 at 2 57 21 PM" src="https://user-images.githubusercontent.com/13592258/80844717-3b1b9f80-8bbc-11ea-983d-d187b28951fc.png">

Reply:
I think adding `-- output` there looks okay, WDYT?
```
INSERT INTO t VALUES ('1');
-- output
  org.apache.spark.sql.AnalysisException: Cannot write incompatible data to table '`default`.`t`':
  - Cannot safely cast 'v': StringType to IntegerType;
```
Could you check documents in the other DBMS-like systems? As the others suggested, I think its better to follow the other document formats.

Reply:
I googled, but didn't have much luck. Only found this one at https://docs.snowflake.com/en/sql-reference/functions/validate.html

<img width="1034" alt="Screen Shot 2020-05-01 at 4 08 33 PM" src="https://user-images.githubusercontent.com/13592258/80847958-64d9c400-8bc6-11ea-9f4b-312750030b40.png">

I personally like to either put ```-- output``` or indent 2 spaces for the error message, or both (the way you suggested)

Reply:
cc: @HyukjinKwon @dongjoon-hyun